autoPyTorch/api/base_task.py

```diff
@@ -771,7 +775,12 @@ def _search(
             disable_file_output (Union[bool, List]):
             load_models (bool), (default=True): Whether to load the
                 models after fitting AutoPyTorch.
-
+            portfolio_selection (str), (default="none"): If "greedy",
+                runs initial configurations present in
+                'autoPyTorch/optimizer/greedy_portfolio.json'.
```

Comment:
Do you think it is better to put it [here](https://github.com/automl/Auto-PyTorch/tree/development/autoPyTorch/configs)?

Reply:
yes. I moved it there